tests/integration.rs

```diff
@@ -40,11 +40,11 @@ where
         .unwrap();
 
     let their_response: Result<Res, _> = pipeline.clone().query_async(&mut their_connection).await;
-
     assert_eq!(our_response, their_response);
 }
 
 #[tokio::test]
+#[serial]
```

Comment:
I think we should run all these tests one after the other and always make sure the databases are clean before each test. That way, commands like `keys` will give more accurate results. In the future, we could also consider using a different DB for each test, though I’m not sure if it’s worth it.

Reply:
I like the idea of a different DB. For Redis side we could use different virtual DBs on the same instance, for Rustdis we could spin up different "instances" on different ports. Wdyt?